lib/scan/cubit/scan_state.dart

```diff
@@ -0,0 +1,30 @@
+part of 'scan_cubit.dart';
```

Comment:
**question**: why did you use the `part of` directive here?

Reply:
To be honest I make my life easy using a VS code to create cubits and blocs 😁

But this indicates this is part of the cubit class :)